src/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "apiURL":"API_URL_NOT_HERE"
```

Comment:
Is this used for testing purposes but overwritten once it is deployed?

Reply:
it needs to be there to be able to do a npm run start and npm run build, but yes will be overwritten when deployed using github action.